packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -214,6 +231,7 @@ tests:
           java: new ObjectId
           csharp: new ObjectId.GenerateNewId
           shell: ObjectId.fromDate
+          php: new \MongoDB\BSON\ObjectId
```

Comment:
How is this used? In Node [ObjectID.createFromTime](https://mongodb.github.io/node-mongodb-native/api-bson-generated/objectid.html#objectid-createfromtime) generates an ObjectId from an existing numeric segment. We have no equivalent constructor but it would be possible to create PHP code to do so.

Reply:
This is a partial test for function call "createFromTime". It consists of two templates "ObjectIdCreateFromTimeTemplate" and "ObjectIdCreateFromTimeArgsTemplate":
````
    ObjectIdCreateFromTimeTemplate: &ObjectIdCreateFromTimeTemplate !!js/function >
        () => {
            return 'new ObjectId';
        }
    ObjectIdCreateFromTimeArgsTemplate: &ObjectIdCreateFromTimeArgsTemplate !!js/function >
        (lhs, arg, isNumber) => {
            if (!isNumber) {
                return `(\\str_pad(\\dechex((${arg})->getTimestamp()), 24, '0'))`;
            }
            return `(\\str_pad(\\dechex(${arg}), 24, '0'))`;
        }
````
As this is only a partial test, only first template is used and as the result it returns only constructor. Examples of full work of these two templates you can find at "bson-methods.yaml".

Reply:
OK to resolve.